src/NuGet.Core/NuGet.Commands/ClientCertificatesCommand/RemoveClientCertRunner.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Linq;
+using NuGet.Common;
+using NuGet.Configuration;
+
+namespace NuGet.Commands
+{
+    public class RemoveClientCertRunner
+    {
+        public static void Run(RemoveClientCertArgs args, Func<ILogger> getLogger)
+        {
+            args.Validate();
+
+            var settings = RunnerHelper.GetSettings(args.Configfile);
+            var clientCertificateProvider = new ClientCertificateProvider(settings);
+
+            var item = clientCertificateProvider.GetClientCertificate(args.PackageSource);
+            if (item == null)
+            {
+                getLogger().LogInformation(string.Format(CultureInfo.CurrentCulture,
+                                                         Strings.NoClientCertificatesMatching,
+                                                         args.PackageSource));
+                return;
+            }
+
+            clientCertificateProvider.Remove(new[] { item });
+
+            getLogger().LogInformation(string.Format(CultureInfo.CurrentCulture, Strings.ClientCertificateSuccessfullyRemoved, args.PackageSource));
```

Comment:
Same behavior have [RemoveSourceRunner](https://github.com/NuGet/NuGet.Client/blob/8cb7886a7e9052308cfa51308f6f901c7caf5004/src/NuGet.Core/NuGet.Commands/SourcesCommands/SourceRunners.cs). Internally this method uses `ISettings.Remove()` which has no successful remove indication.

Reply:
Tried to repro by opening the config file with another program. In that case I got an early error (edited):

```powershell
PS C:\repos\NuGet.Client> .\artifacts\VS15\NuGet.exe client-certs remove -packagesource bbbbbbbbb
Unexpected failure reading NuGet.Config. Path: '%APPDATA%\NuGet\NuGet.Config'.
  The process cannot access the file '%APPDATA%\NuGet\NuGet.Config' because it is being used by another process.
```

Hence, it may not be helpful to check this case. For now, I will resolve this conversation.